mobile/src/main/java/org/openhab/habdroid/ui/OpenHABWidgetAdapter.java

```diff
@@ -512,6 +519,15 @@ public void onStopTrackingTouch(SeekBar seekBar) {
             Util.sendItemCommand(mConnection.getAsyncHttpClient(), mBoundItem,
                     String.valueOf(seekBar.getProgress()));
         }
+
+        @Override
+        public boolean onTouch(View v, MotionEvent motionEvent) {
+            if (motionEvent.getActionMasked() == MotionEvent.ACTION_UP) {
+                Util.sendItemCommand(mConnection.getAsyncHttpClient(), mBoundItem,
+                        mSliderSwitch.isChecked() ? "0" : "100");
```

Comment:
The feature request explicitly asked for this *not* being a shortcut to 0/100. Sending ON/OFF would be appropriate here IMHO.

Reply:
Right, changed it to ON/OFF